oaknational/Oak-Web-Application
# fix(cms-image): fix aspect-ratio when cropRect passed

## Description

- change so that now the aspect-ratio of the cropping rect is respected, in the case when `cropRect` is passed

## How to test

No components currently use it, but check [the video here](https://www.notion.so/oaknationalacademy/How-to-use-images-in-OWA-7c363eebcbab4111932fff5355adf23c) -- in the section about `cropRect` you can see that aspect ratio is preserved
